qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/diagnostics/QuteErrorCode.java

```diff
@@ -78,6 +78,12 @@ public enum QuteErrorCode implements IQuteErrorCode {
 
 	UndefinedSectionTag("No section helper found for `{0}`."), //
 
+	UndefinedUserTagParameter("No parameter `{0}` found for `{1}` user tag."), //
+	
+	TooManyUserTagItParameter("Too many `it` parameter declared in `{0}` user tag."), //

```

Comment:
What do you think about adding an error for repeat parameter declarations? i.e.

```qute-html
{#myTag title="The Title" title="The Other Title"}
  <ul>
    <li>content</li>
  </ul>
{/myTag}
```

it would be nice if there was a warning for the second `title` parameter.

Reply:
Great idea! It should be fixed:

![image](https://user-images.githubusercontent.com/1932211/218541108-67f97e83-c729-4b69-ba9a-1c0454cb7946.png)